src/LogicApp/LogicApp/Cmdlets/LogicApp/GetAzureLogicAppRunHistoryCommand.cs

```diff
@@ -45,6 +46,15 @@ public class AzureLogicAppRunHistoryCommand : LogicAppBaseCmdlet
         [ValidateNotNullOrEmpty]
         public string RunName { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Indicates the cmdlet should follow relation links.")]
```

Comment:
Why not "next page links"? The actual property is "NextPageLink"
Same for the variable name, it should be consistent with the actual property followed.